templates/sysv/lsb-3.1/init.sh

```diff
@@ -88,7 +88,13 @@ stop() {
       sleep 1
     done
     if status ; then
-      emit "$name stop failed; still running."
+      emit "$name stop failed; still running.  Killing with SIGKILL"
+      trace "Killing $name (pid $pid) with SIGKILL"
+      kill -KILL $pid
```

Comment:
Where would this setting go?  I haven't trolled through all of the code here, so I'm asking to shorten my dev cycle :)